src/System.CommandLine/IdentifierSymbol.cs

```diff
@@ -60,9 +60,12 @@ public override string Name
         }
 
         /// <summary>
-        /// Adds an alias. Multiple aliases can be added, most often used to provide a shorthand alternative.
+        /// Adds an <see href="/dotnet/standard/commandline/syntax#aliases">alias</see>.
         /// </summary>
         /// <param name="alias">The alias to add.</param>
+        /// <remarks>
+        /// You can add multiple aliases for a symbol.
```

Comment:
I thought there was some info added about what aliases could be used for. Not serious though.

Reply:
I replaced that in the summary with a link on "alias" to the aliases section of the syntax article https://docs.microsoft.com/dotnet/standard/commandline/syntax#aliases